python-backend/app/config.py

```diff
@@ -57,6 +58,8 @@ class Config(object):
     # 100MB file limit
     MAX_CONTENT_LENGTH = 100 * 1024 * 1024
 
+    SCHEDULER_API_ENABLED = True
```

Comment:
If this flag is what controls for the scheduler to run, can we just set this to false in dev and test and True in Prod?
Would that save us from the hassle of checking the environment variable all over the place?

Reply:
When I was first adding the scheduler I brought this over thinking it was needed but this just adds endpoints to the app that allow for api scheduling which I don't like and have now set that to false.